internal/campaigns/util.go

```diff
@@ -15,14 +15,26 @@ import (
 	yamlv3 "gopkg.in/yaml.v3"
 )
 
+type CodehostCapability string
+
+const (
+	CodehostCapabilityLabels          CodehostCapability = "Labels"
+	CodehostCapabilityDraftChangesets CodehostCapability = "DraftChangesets"
+)
+
+type CodehostCapabilities struct {
```

Comment:
How about something like this?

```suggestion
type CodehostCapabilities map[CodehostCapability]bool
```